index.bs

```diff
@@ -8440,6 +8440,48 @@ corresponding to [=interface members=].
 </div>
 
 
+<h4 id="InNamespace" extended-attribute lt="InNamespace">[InNamespace]</h4>
+
+If the [{{InNamespace}}] [=extended attribute=] appears on an [=interface=], it indicates that
+the [=interface object=] for this interface will not be created as a property of the global
+object, but rather the namespace which is the argument to the extended attribute.
+
+The [{{InNamespace}}] extended attribute [=takes an identifier|take an identifier=].
+This identifier must be the identifier of a namespace.
+
+The [{{InNamespace}}] and [{{NoInterfaceObject}}]
+extended attributes must not be specified on the same interface.
+
+The [{{InNamespace}}] extended attribute
+must not be used on a [=callback interface=].
+
+The [=class string=] of an interface with the [{{InNamespace}}] extended
+attribute is the concatenation of the class string of the namespace, <code>"."</code>,
+and the class string that the interface would otherwise have without this
+extended attribute.
```

Comment:
I think this should not be located here but instead be a modification to

> The class string of a platform object that implements one or more interfaces must be the identifier of the primary interface of the platform object.

Also, no `<code>` around strings in Web IDL, it appears.

Reply:
Done